### ISSUE TITLE
Rename to Nix Foundation

### ISSUE THREAD

**Issue by fricklerhandwerk** [NixOS/foundation#77]
The foundation and community it is serving is clearly not about just NixOS. Nix is the cornerstone of the ecosystem.

Related, and more context why using appropriate names would help with reducing confusion: https://github.com/NixOS/org/issues/80

I understand that renaming a public entity is associated with costs. This issue is mostly for reference, since the topic came up quite often in the past months.

**Comment by zimbatm** [comment]
Let's address the fundamental problem; NixOS refers both to the project and the Linux distribution. It's that overlap of terms that is confusing.

Renaming the project would require changing the GitHub Org, domain and foundation. How about we rename the Linux distribution instead?

**Comment by infinisil** [comment]
@zimbatm I don't think that makes sense, "OS" clearly refers to an operating system. And if NixOS is the project, then should the language be called "The NixOS language"? And should we change the CLI to `nixos build`, etc.? That would be very confusing and unintuitive on all levels.

Renaming the project to just "Nix" makes the most sense.

**Comment by zimbatm** [comment]
Hold on, you're taking what I said and running away with it; I didn't say we should rename Nix either. Nix is also an overloaded term because it refers to the language, implementation, build system, ...

If you can forget for a second that "OS" maps to the operating system, it's possible to attribute another meaning. Do you still think about the fruit when talking about Apple computers? What I am saying is that if we accept this as a historical accident, then it opens new possibilities.

Eg:
* Nixos - the project / GitHub org / domain
* Nixos Foundation - the foundation that represents the project
* Nix Distri - the Linux distribution. Or find another name, I don't care.
* Nixexpr - the language
* CNix - the implementation
* ...

**Comment by zmitchell** [comment]
> Let's address the fundamental problem; NixOS refers both to the project and the Linux distribution. It's that overlap of terms that is confusing.
> 
> Renaming the project would require changing the GitHub Org, domain and foundation. How about we rename the Linux distribution instead?

We would have Nix the language, `nix` the package manager and build system, Nix (?) the OS, NixOS the abstract project, and the NixOS Foundation? That seems worse than what we have now where at least the name of the project, the OS, and the foundation all have NixOS in the name.

I’m not sure this is the most pressing issue right now, but if the name is going to change I agree that Nix Foundation is better since Nix is the common denominator.

**Comment by KFearsoff** [comment]
Here's what I got when I made my research:

- There isn't consistent naming for the community and its activities. Looking at the [community page](https://nixos.org/community/), we can see the following used:
  - The website itself is under domain nixos.org (implying that related organization/community is NixOS)
  - "Nix community" is used to describe the whole community
  - The official Matrix space is also hosted under nixos.org
  - Mastodon and Twitter, which are described as "official spaces", are also tagged under NixOS
  - Stackoverflow link to "official space" looks for tags: "Nix", "Nixpkgs", "NixOS", "NixOps"
  - NixCon is named after Nix, not NixOS
  - The link under "RFC process" leads to the official organization on Github, which is called NixOS
  - Under the "foundation board" entry, "NixOS project" is mentioned
  - Under the "security team" entry, "NixOS community" is mentioned
  - Under the "infrastructure team", both "Nix community" and "NixOS community" are mentioned; likewise for "NixCon team"
  - Under the "marketing team", "Nix user base" is mentioned
  - Under the "CUDA team", both "Nix" and "NixOS" are mentioned
  - There's a "Nix team", which is responsible for Nix package manager specifically; makes sense
  - There's a "Nix formatting team", which creates a standard format for "Nix". It is ambiguous if the team creates a standard format for Nix package manager or for Nix DSL
  - NixOS Foundation is mentioned, which once again suggests that the project is NixOS
  
  I think there are three important questions we have to ask in order to get to the "Nix" lowest common denominator:
  
  - What do we do about the domain name? This is addressed by this, not-yet-published RFC: https://github.com/nix-rfc-canonical-domain/rfcs
  - What do we do with NixOS Foundation, specifically? AFAIK, it is a legal entity. Can we rename it without much issue? Do we just leave the legal name as-is, and rename the "community name" to Nix Foundation?
  - What about our Github organization? Can we rename it to `nix-project` or something, is it worth it, and what are the downsides?
